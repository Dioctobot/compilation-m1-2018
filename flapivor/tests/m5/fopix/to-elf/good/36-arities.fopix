def f0 () = 0
def f1 (x) = 1
def f2 (x, y) = 2
def f3 (x, y, z) = 3
def f4 (x, y, z, t) = 4
def f5 (x, y, z, t, u) = 5
def f6 (x, y, z, t, u, v) = 6
def f7 (x, y, z, t, u, v, w) = 7

val main =
    print_int (f0 () + f1 (0) + f2 (0, 1) + f3 (0, 1, 2) + f4 (0, 1, 2, 3) + f5 (0, 1, 2, 3, 4)
               + f6 (0, 1, 2, 3, 4, 5) + f7 (0, 1, 2, 3, 4, 5, 6))