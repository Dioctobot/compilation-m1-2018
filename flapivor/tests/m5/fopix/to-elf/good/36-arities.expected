Process exited with status exited(0).
STDOUT:
28
STDERR:



